static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -154,66 +145,21 @@ const handlePlay = (atomId: string, player: AtomPlayer): void => {
     }
 };
 
-const getYoutubeIdFromUrl = (url: string): string => {
-    const youtubeIdKey = 'v';
-    const splitUrl = url.split('?');
-
-    if (splitUrl.length === 1) {
-        return '';
-    }
-
-    const queryParams = getUrlVars(splitUrl[1]);
-
-    return queryParams[youtubeIdKey] || '';
-};
-
-const onPlayerPlaying = (atomId: string): void => {
-    const player = players[atomId];
+const onPlayerPlaying = (uniqueAtomId: string): void => {
+    const player = players[uniqueAtomId];
 
     if (!player) {
         return;
     }
 
-    const { youtubePlayer, youtubeId } = players[atomId];
-
-    /**
-     * Get the youtube video id from the video currently playing.
-     * We want to compare with the youtube ID in memory. If they differ
-     * a related video has begun playing, so we need to get the atom ID
-     * for tracking.
-     */
-    const latestYoutubeId = getYoutubeIdFromUrl(youtubePlayer.getVideoUrl());
+    const { youtubePlayer, atomId } = players[uniqueAtomId];
 
-    if (youtubeId !== latestYoutubeId) {
```

Comment:
🤔 i remember wondering how that would be possible? @akash1810 ?

Reply:
After discussing with Akash, it seems that whole logic was used for an AB test and shouldn't be necessary anymore. Can you confirm, @GHaberis or @nicl or @philmcmahon ? 😄

Reply:
@regiskuckaertz this is still necessary, `youtubeId` and `latestYoutubeId` can become unequal once a user starts playing a related video from the videos presented to them using youtube's native "related videos" feature.

Reply:
From memory this was used during the A/B test. That is, success of the test to move to the standard yt endslate was to track onward journeys using video starts as the metric. Now that the test is complete and the yt endslate won, I don't think we need to continue this tracking?

Reply:
>  I don't think we need to continue this tracking?

We'd need to confirm this with Matt Ho. The tracking this enables doesn't just cover video starts, but also the additional milestones (25, 50, 75% played etc), if we stop tracking this then video plays triggered from youtube's endslate would not appear in our own tracking data.

Reply:
I feel like this is a discussion for another PR. I'll speak with Matt, thanks both for your help 👍